packages/desktop-client/src/components/common/Card.tsx

```diff
@@ -17,8 +17,8 @@ const Card = forwardRef<HTMLDivElement, CardProps>(
           marginLeft: 5,
           marginRight: 5,
           borderRadius: 6,
-          backgroundColor: 'white',
-          borderColor: colors.p3,
+          backgroundColor: theme.cardBackground,
+          borderColor: theme.cardBorder,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this is a different color